src/hotspot/share/opto/macro.cpp

```diff
@@ -816,6 +818,16 @@ SafePointScalarObjectNode* PhaseMacroExpand::create_scalarized_object_descriptio
     const TypeOopPtr *field_addr_type = res_type->add_offset(offset)->isa_oopptr();
 
     Node *field_val = value_from_mem(sfpt->memory(), sfpt->control(), basic_elem_type, field_type, field_addr_type, alloc);
+    // If scalarize operation is adding too many nodes, bail out
+    /*if (C->live_nodes() + NodeLimitFudgeFactor > C->max_node_limit()) {
+      if (C->do_escape_analysis() == true && !C->failing()) {
+        // Retry compilation without escape analysis.
+        // If this is the first failure, the sentinel string will "stick"
+        // to the Compile object, and the C2Compiler will see it and retry.
+        C->record_failure(C->congraph()->invocation() > 0 ? C2Compiler::retry_no_iterative_escape_analysis() : C2Compiler::retry_no_escape_analysis());
+      }
+      return nullptr;
+    }*/
```

Comment:
Ah, it looks like you commented out your fix. That would explain why the GitHub Actions tests are failing ;)